kamaki/clients/cyclades/__init__.py

```diff
@@ -44,7 +44,7 @@ class CycladesComputeClient(CycladesComputeRestClient, Waiter):
     CONSOLE_TYPES = ('vnc', 'vnc-ws', 'vnc-wss')
 
     def create_server(
-            self, name, flavor_id, image_id,
+            self, name, flavor_id, image_id, key_name,
```

Comment:
maybe `key_name=None`? key_name is optional. Also, you probably need to also add the paramameter to `clients.compute.create_server`.